src/reader.rs

```diff
@@ -930,6 +1031,64 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_read_link_type_link() {
+        let kml_str = r#"<Link id="Some ID">
+            <href>/path/to/local/resource</href>
+            <refreshMode>onChange</refreshMode>
+            <refreshInterval>4</refreshInterval>
+            <viewRefreshMode>onStop</viewRefreshMode>
+            <viewRefreshTime>4</viewRefreshTime>
+            <viewBoundScale>1</viewBoundScale>
+            <viewFormat></viewFormat>
+        </Link>"#;
+
+        let mut attrs = HashMap::new();
+        attrs.insert("id".to_string(), "Some ID".to_string());
+
+        let l: Kml = kml_str.parse().unwrap();
+        assert_eq!(
+            l,
+            Kml::LinkTypeLink(LinkTypeLink {
+                href: Some("/path/to/local/resource".to_string()),
+                refresh_mode: Some(types::RefreshMode::OnChange),
+                view_refresh_mode: Some(types::ViewRefreshMode::OnStop),
+                view_format: Some(String::new()),
+                attrs,
```

Comment:
Do we want to store the attributes? I see some other types do, but most ignore them. The spec only allows three here: `id`, `targetId`, and a user-defined one. Maybe I should ignore them to conform to how it is done for other types?

As an aside, I could see how _many_ attributes could be an issue with memory. Perhaps that's a separate issue to be solved in another PR.

Reply:
I think enough of the other types include `attrs` to keep it for now, but worth flagging for the future